common/djangoapps/student/tests/test_login.py

```diff
@@ -460,8 +460,11 @@ def test_externalauth_login_required_course_context(self):
         noshib_response = self.client.get(TARGET_URL, follow=True)
         self.assertEqual(noshib_response.redirect_chain[-1],
                          ('http://testserver/login?next={url}'.format(url=TARGET_URL), 302))
-        self.assertContains(noshib_response, ("Sign in or Register | {platform_name}"
-                                              .format(platform_name=settings.PLATFORM_NAME)))
+        self.assertContains(
```

Comment:
Are we just being more specific with validation here, or has the actual response changed?  If it has changed we need to be confident that we have not caused problems for any clients.